File: src/main/java/me/geekTicket/Command/CommandCore.java
package me.geekTicket.Command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

public class CommandCore implements CommandExecutor {

    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {

        if (args.length == 0 || args[0].equalsIgnoreCase("help")) {
            CommandHelp.execute(sender, command, label, args);
            return true;
        }
        if (args[0].equalsIgnoreCase("give")) {
            CommandGive.execute(sender, command, label, args);
        }
        if (args[0].equalsIgnoreCase("take")) {
            CommandTake.execute(sender, command, label, args);
        }

     return true;
    }
}

File: src/main/kotlin/me/geekTicket/Command/CommandCore.kt
package me.geekTicket.Command


import me.geekTicket.Bukkit.LangMain
import org.bukkit.command.CommandSender
import taboolib.common.platform.command.CommandBody
import taboolib.common.platform.command.CommandHeader
import taboolib.common.platform.command.mainCommand

@CommandHeader(name = "geekt", aliases = ["gkt", "gekt"])
object CommandCore {
    @CommandBody(permission = "geekt.command.give", optional = true)
    val give = CommandGive.command

    @CommandBody(permission = "geekt.command.take", optional = true)
    val take = CommandTake.command

    @CommandBody(permission = "geekt.command.set", optional = true)
    val set = CommandSet.command

    @CommandBody(optional = true)
    val help = CommandHelp.command

    @CommandBody(permission = "geekt.command.reload", optional = true)
    val load = CommandReload.command

    @CommandBody(permission = "geekt.admin", optional = true)
    val taskrun = CommandTaskRun.command


    @CommandBody
    val main = mainCommand {
        execute<CommandSender> { sender, _, argument ->
            if (argument.isEmpty()) {
                sender.sendMessage("§lGeekTicket §8- §7§l帮助导航 §8- §f§lBate")
                for (list_to_string in LangMain.HELP) {
                    sender.sendMessage(list_to_string)
                }
                return@execute
            }
            sender.sendMessage("§8[§3§lGeekTicket§8] §cERROR §3| 未知的参数 §6$argument")
        }
    }
}